docs/SOP.md

```diff
@@ -18,19 +18,19 @@ This document contains standard operating procedures (SOPs) for the OBO Foundry
 
 ### New Ontology Requests (NOR)
 
-1. When receiving a new ontology request (NOR), the OBO dashboard administrator should thank the submitter for their submission.
```

Comment:
@erik-whiting note that the repetition of `1.` is potentially nice since it reduces diff if there is ever an additional bullet added in the middle. Markdown is smart and knows to number appropriately